py3-sphinx.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: py3-sphinx
   version: "8.2.3"
-  epoch: 0
+  epoch: 1
```

Comment:
thx. but that begs the question, why bump it at all now?

Reply:
fair.

Updated.